src/index.css

```diff
@@ -1,13 +1,66 @@
-body {
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 16px;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.navbar {
+  width: 96%;
+  border: 2px grey solid;
+  padding: 10px;
+  margin: 0 2% 5px 2%;
+}
+
+.dropdown_button {
+  background-color: #fff;
+  border: none;
+  padding: 5px;
+}
+
+.navbar li {
+  display: inline;
+  border: aquamarine 1px solid;
+  padding: 5px;
+  margin: 12px 0 12px 20%;
+}
+
+.dropdown_list {
+  margin-top: 1rem;
+}
+
+.header {
```

Comment:
I noticed that both `header` and `todo_item` selectors shared the same property:
```css
color: rgb(5, 113, 113);
```

You can consider using **comma-separated** list to apply the same styles to multiple selectors and thus avoid repetition 👍🏼.